Tests/SpeziDevicesTests/HealthMeasurementsTests.swift

```diff
@@ -10,19 +10,21 @@ import HealthKit
 @_spi(TestingSupport) import SpeziBluetooth
 import SpeziBluetoothServices
 @_spi(TestingSupport) @testable import SpeziDevices
-import XCTest
+import Testing
 
 
-final class HealthMeasurementsTests: XCTestCase {
+@Suite
+final class HealthMeasurementsTests {
     @MainActor
+    @Test
```

Comment:
as part of switching to Swift Testing, we should remove the `@MainActor` isolation from all the individual test functions, and instead simply isolate the type as a whole to the main actor (iirc there was some XCTest-related reason for why we didn't do it this way in the first place)